lib/screens/home/explorer/proposals_flow/proposal_weighted_vote_confirm/proposal_weighted_vote_confirm.dart

```diff
@@ -0,0 +1,214 @@
+import 'package:provenance_dart/proto_gov.dart' as gov;
+import 'package:provenance_wallet/common/pw_design.dart';
+import 'package:provenance_wallet/common/widgets/button.dart';
+import 'package:provenance_wallet/common/widgets/modal_loading.dart';
+import 'package:provenance_wallet/common/widgets/pw_list_divider.dart';
+import 'package:provenance_wallet/common/widgets/pw_slider.dart';
+import 'package:provenance_wallet/dialogs/error_dialog.dart';
+import 'package:provenance_wallet/screens/home/explorer/proposals_flow/proposal_weighted_vote/weighted_vote_bloc.dart';
+import 'package:provenance_wallet/screens/home/explorer/proposals_flow/proposal_weighted_vote_confirm/weighted_vote_option_column.dart';
+import 'package:provenance_wallet/screens/home/explorer/proposals_flow/proposals_flow_bloc.dart';
+import 'package:provenance_wallet/screens/home/transactions/details_item.dart';
+import 'package:provenance_wallet/services/models/account.dart';
+import 'package:provenance_wallet/services/models/proposal.dart';
+import 'package:provenance_wallet/util/get.dart';
+import 'package:provenance_wallet/util/strings.dart';
+
+class ProposalWeightedVoteConfirmScreen extends StatefulWidget {
+  const ProposalWeightedVoteConfirmScreen({
+    Key? key,
+    required this.account,
+    required this.proposal,
+  }) : super(key: key);
+
+  final TransactableAccount account;
+  final Proposal proposal;
+
+  @override
+  State<StatefulWidget> createState() =>
+      _ProposalWeightedVoteConfirmScreenState();
+}
+
+class _ProposalWeightedVoteConfirmScreenState
+    extends State<ProposalWeightedVoteConfirmScreen> {
+  double _gasEstimate = 1.25;
```

Comment:
Is this value copied from somewhere? Wondering if all usages should be using the same `const` or if multiple constants could be in one file.

Reply:
Yes, it's the standard default for gas estimates and it's a value that's repeated over and over and over. I'll find a place to put it so we don't have this magic number everywhere.